cmd/loadtest/recall.go

```diff
@@ -17,7 +17,9 @@ func getRecentBlocks(ctx context.Context, ec *ethclient.Client, c *ethrpc.Client
 	if err != nil {
 		return nil, err
 	}
-	rawBlocks, err := util.GetBlockRange(ctx, bn-*inputLoadTestParams.RecallLength, bn, c)
+
+	// FIXME the batch size of 25 is hard coded and probably should at least be a constant or a parameter
+	rawBlocks, err := util.GetBlockRangeInPages(ctx, bn-*inputLoadTestParams.RecallLength, bn, 25, c)
```

Comment:
You can use `*ltp.BatchSize` in `func mainLoop(ctx context.Context, c *ethclient.Client, rpc *ethrpc.Client) error {}` to set the batch size instead of hardcoding a value.